docs/private-app-manage.md

```diff
@@ -0,0 +1,22 @@
+## How to manage Private Apps 
+
+### List Private Apps
+
+Via the Private App management page, click on the Manage Private Apps button (Fig 1) to see a list of enabled and disabled Private Apps . 
```

Comment:
Please remove extra space before full stop

Reply:
Done, in next merge